### ISSUE TITLE
Holding shift while using rotating input should stop every 45 degree intervals

### ISSUE THREAD

**Issue by TheColaber** [ScratchAddons/ScratchAddons#3699]
https://discord.com/channels/806602307750985799/806609030935740477/891288358800736276
> shift + rotating this to only stop at 0, 45, 90, 135, 180, -135, -90, -45
![image](https://user-images.githubusercontent.com/72760579/138565314-4e3060c1-64db-41d8-86f8-d4519edf4b11.png)
 
useful

**Comment by Norbiros** [comment]
I think we need to close it due to #3726
`WL send: Closing. There's already a number input to set the angle, and typing is faster.`

**Comment by WorldLanguages** [comment]
Yeah, see here, in both types of rotating inputs (Scratch has 2 of them for some reason) you can just type 45, 90, 135, 180, etc. manually.

![image](https://user-images.githubusercontent.com/17484114/153665390-fa109b0e-785a-44ff-b3a8-b2f21c40d723.png)

**Comment by RedGuy12** [comment]
and on the left one, it already snaps

**Comment by WorldLanguages** [comment]
> and on the left one, it already snaps

every 15 degrees

**Comment by Samq64** [comment]
At least one person suggested it, so this should be re-considered.

See #5809 and https://github.com/ScratchAddons/ScratchAddons/pull/3726#issuecomment-1473674556

**Comment by DNin01** [comment]
I just created a new issue that takes on a different approach to this idea.
